### DISCUSSION TITLE
Link to panel from an alert in the unified alert list

### DISCUSSION THREAD

**Discussion by mathgaming** [grafana/grafana#62576] [Alerting]
When you press an alert in the old alert list it goes to the panel.
I would like to have this feature in the new unified alert list

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is being addressed here https://github.com/grafana/grafana/issues/50691. If I am mistaken, feel free to make an [issue](https://github.com/grafana/grafana/issues) to would like to request the feature.
